build/code-batch-process.go

```diff
@@ -229,9 +229,9 @@ func containsString(a []string, s string) bool {
 	return false
 }
 
-func giteaFormatGoImports(files []string) error {
+func giteaFormatGoImports(files []string, changedFiles, writeFile, outputDiff bool) error {
```

Comment:
```suggestion
func giteaFormatGoImports(files []string, changedFiles bool, writeFile bool, outputDiff bool) error {
```

Might be good to enforce type in func arguments actually. Also, these vars seem a bit inappropriately named to me (would call them `hasChangedFiles`, `doWriteFile`, `doOutputDiff`).

Reply:
Their type already is enforced. That's what the bool at the end does.